plugins/com.google.cloud.tools.eclipse.appengine.libraries/src/com/google/cloud/tools/eclipse/appengine/libraries/model/MavenCoordinates.java

```diff
@@ -16,9 +16,10 @@
 
 package com.google.cloud.tools.eclipse.appengine.libraries.model;
 
-import com.google.common.base.Preconditions;
 import java.text.MessageFormat;
```

Comment:
Looks like your import rules are out of sync: Google style now just sorts them as a block with no special priority on java.* or com.google.*

Reply:
Yes, that's true. I don't think we've updated the external Google stylesheet for Eclipse but let me check.

Yep. Still uses the old import ordering: https://github.com/google/styleguide/issues/270